frontend/src/views/ComponentsView.svelte

```diff
@@ -95,7 +105,7 @@
   hr {
     border: 1px dashed var(--main-fg-color);
     width: 100%;
-    margin: 2em 0;
+    margin: 0.5em 0;
```

Comment:
```suggestion
    margin: 1em 0;
```

Let's split the difference and bump it up a little.